### PR TITLE
Fixed Datepicker sample

### PR DESCRIPTION
**Pull request by battletoilet**
This was really an issue with how we pulled in scripts onto the templates. The gulp task was only setup to include one script and since pickate has two it was breaking pickadate. Now the gulp task should handle an infinite number of scripts that are placed in the src/component/ folders.

### CODE REVIEW

File: gulp/fabric.js [hunk 1]
```diff
@@ -517,7 +520,9 @@ gulp.task('component-samples-template', ['build-component-data', 'component-samp
             .on('error', onGulpError)
         .pipe(data(function () {
             var jstag = '';
-            if(typeof storedFiles[folder].script != "undefined") { jstag = storedFiles[folder].script; }
+            var files =  storedFiles[folder]["files"];
```
Comment (notandrew) [inline gulp/fabric.js]: there's an extra space before 'storedFiles'

File: gulp/fabric.js [hunk 2]
```diff
@@ -517,7 +520,9 @@ gulp.task('component-samples-template', ['build-component-data', 'component-samp
...
+            for(var o=0; o < files.length; o++) {jstag += files[o];}
+            if(typeof storedFiles[folder][folder] != "undefined") { jstag += storedFiles[folder][folder]; }
```
Comment (notandrew) [inline gulp/fabric.js]: Would you mind adding some spaces to the for, if, and brackets?  Also, new lines for the jstag lines, please.  :)

**Comment by notandrew** [comment]
looks good to me

**Comment by wdo3650** [comment]
This looks good to me. Nice work!

**Pull request merged**